webpack.config.js

```diff
@@ -1,7 +1,6 @@
 const webpack = require( 'webpack' ),
 	path = require( 'path' ),
-	ExtractTextPlugin = require( 'extract-text-webpack-plugin' ),
```

Comment:
In most of our code, we we never use this comma-separated `const` syntax. Let's stick with the same style everywhere.